spec/models/measure_spec.rb

```diff
@@ -1232,8 +1248,45 @@
       it { expect(measure.units).to eq(expected_units) }
     end
 
-    context 'when there are just measure conditions' do
```

Comment:
Maybe we can keep the context 'when there are just measure conditions' ?
so we can keep just a single ' subject(:measure)' ? 

Nitpick: can we use "only" instead of "just" ... it sounds better 😄

Reply:
Yeah, I can do that for sure :)